fish_prompt.fish

```diff
@@ -407,6 +406,17 @@ function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   set_color normal
 end
 
+function __bobthefish_prompt_rubies -d 'Display current Ruby information'
+  switch "$theme_display_ruby"
+    case no; return
+    case auto
```

Comment:
What exactly are you trying to accomplish by only showing the ruby version inside the root directory affected by a version switch? If you run `ruby` inside a subdirectory, it'll be affected by the `.ruby-version` as well…